Added new P15 Tier for Managed Disks

bahreex opened the pull request:
Added new P15 tier for Managed Disks in the corresponding table, which now supports 256 GB Disk Size.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

bahreex committed:
Added new P15 Tier for Managed Disks

Added new P15 tier for Managed Disks in the corresponding table, which now supports 256 GB Disk Size.
